ibis/expr/types/generic.py

```diff
@@ -2684,6 +2684,24 @@ def nth(self, n: int | ir.IntegerValue) -> Column:
         """
         return ops.NthValue(self, n).to_expr()
 
+    def to_list(self, **kwargs) -> list:
+        """Convert a column expression to a list.
+
+        Parameters
+        ----------
+        kwargs
+            Same as keyword arguments to [`to_pyarrow`](#ibis.expr.types.core.Expr.to_pyarrow)
```

Comment:
This is technically a broken link, since the docs aren't published... but so is

```
[`execute`]((#ibis.expr.types.core.Expr.execute)
```

used elsewhere.

Reply:
We have a link checker -- maybe it doesn't work on quartodoc-generated links?  But yeah, `lychee` is the link checker and there's a `checklinks` rule in the justfile, so I dunno.

Reply:
Ah, interesting, we do publish `Column` docs, but not inherited methods, so we're missing out on things like `to_pyarrow`.

Well, that's a separate concern